dubbo-registry/dubbo-registry-nacos/src/main/java/org/apache/dubbo/registry/nacos/NacosRegistry.java

```diff
@@ -369,15 +370,26 @@ private void filterServiceNames(Set<String> serviceNames, URL url) {
         return serviceNames;
     }
 
-    private List<URL> buildURLs(URL consumerURL, Collection<Instance> instances) {
-        if (instances.isEmpty()) {
-            return Collections.emptyList();
+    private List<URL> toUrlWithEmpty(URL consumerURL, Collection<Instance> instances) {
```

Comment:
Last nitpick, what about changing  `toUrlWithEmpty` to  `toUrlsWhenEmpty`? (a suggestion, there should be another better method name to make it clear)

I know we use `toUrlWithEmpty` in zookeeperRegistry.class, but I found the name may bring us a little confusion :)

Reply:
The name tells that the protocol will be set "empty"